manifests/xgboost-operator/base/cluster-role-binding.yaml

```diff
@@ -1,11 +1,13 @@
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Why downgrading this?

Reply:
Sorry, that's a mistake. But I did downgrade for deployment yaml, cause when I do the installation, there's a error about deployment version, after I downgrade it, everything work fine

Reply:
What error is it?

Reply:
error: unable to recognize "manifests/xgboost-operator/base/deployment.yaml": no matches for kind "Deployment" in version "extensions/v1beta1"
After I downgrade to apps/v1 , error disappear.
I also checked deployment.yaml on mpi operator, the version is appa/v1 and works fine